util.py

```diff
@@ -55,12 +55,16 @@ def copyFilesFromDir(sourcedir, dest):
 
 def runCmd2(command, with_stdout=False, with_stderr=False, inputtext=None):
 
-    cmd = subprocess.Popen(command, bufsize=1,
-                           stdin=(inputtext and subprocess.PIPE or None),
-                           stdout=subprocess.PIPE,
-                           stderr=subprocess.PIPE,
-                           shell=isinstance(command, str),
-                           close_fds=True)
+    try:
+        cmd = subprocess.Popen(command, bufsize=1,
+                               stdin=(inputtext and subprocess.PIPE or None),
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.PIPE,
+                               shell=isinstance(command, str),
+                               close_fds=True)
+    except ex:
```

Comment:
I think this should include the call to communicate() since that involves reading, writing, and waiting and it could fail for various reasons.

Reply:
Makes sense. I'd also get rid of the commented-out block in the middle, since we definitely don't want to use that buggy code (a `poll()` could do if needed, anyway), but likely in a preliminary patch, then.